src/services/cache.service.ts

```diff
@@ -0,0 +1,30 @@
+import {createClient} from 'redis';
+
+export const DEFAULT_TIME_TO_LIVE=240;
+
+const redisClient = createClient();
+
+redisClient.connect();
+
+redisClient.on('connect',()=>{
+    console.log('[REDIS] : connected!');
+});
+
+redisClient.on('error',(error)=>{
+    console.log(`[REDIS] : error ${error}`);
+});
+
+export const setCache = async (key:string,toBeCachedValue:string,ttl:number):Promise<string>=> {
```

Comment:
```suggestion
export const setCache = async (key:string,value: string,ttl:number):Promise<string>=> {
```

you can just call it value.